plugins/modules/keycloak_realm_key.py

```diff
@@ -119,10 +117,10 @@
 notes:
   - Current value of the private key cannot be fetched from Keycloak. Therefore comparing its desired state to the current
     state is not possible.
-  - If certificate is not explicitly provided it will be dynamically created by Keycloak. Therefore comparing the current
+  - If O(config.certificate) is not explicitly provided it is dynamically created by Keycloak. Therefore comparing the current
     state of the certificate to the desired state (which may be empty) is not possible.
   - Due to the private key and certificate options the module is B(not fully idempotent). You can use O(force=true) to force
-    the module to always update if you know that the private key might have changed.
+    the module to ensure update if you know that the private key might have changed.
```

Comment:
Hmm, to me `always` sounded better. `ensure update` sounds strange to me - but then I'm not a native English speaker, so :shrug: